src/main/java/com/easyrest/components/BaseComponent.java

```diff
@@ -0,0 +1,18 @@
+package com.easyrest.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class BaseComponent {
+    protected WebDriver driver;
+
+    public BaseComponent(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public void enterData(String data, By element, WebDriver driver){
```

Comment:
I would named that method sendKeys and variable key instead of data.

Maybe there is one more way to implement this and not to pass driver as parameter. If we create object of baseComponent in needed class and call method that we need, will it be correct? Or make BaseComponent abstract or method static?